src/views/fightSimulator.vue

```diff
@@ -946,6 +960,29 @@ export default {
 
         },
 
+        displayFightOutcome(){
+            let text = document.getElementById('Difference');
+            let value = parseFloat(text.innerHTML);
+            if (value > 1){
+                text.style.backgroundColor = "lightgreen";
+            }
+            if (value < 1){
+                text.style.backgroundColor = "IndianRed";
+            }
+            if (value === 0 || value === 1){
+                text.style.backgroundColor = "white";
+            }
+
+        },
+
+        //format numbers
+        format(x){
+            return x.toString().replace(/\B(?<!\.\d*)(?=(\d{3})+(?!\d))/g, ",");
+        },
+
+
+
+        //not used anymore
```

Comment:
can be removed if not used, or is there a reason to keep them (maybe for later use or so)?

same for the next two methods

Reply:
I want to keep them for potential future use